### PR TITLE
storage: copy guest image from data dir to nfs share before cloning

### PR DESCRIPTION
**Pull request by balamuruhans**
cloning of VM fails if test expects vm image to be available in
nfs share location, so check for vm image bootstrapped in data
directory and copy it.

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by balamuruhans** [comment]
**Guest import from NFS share location:**

```
2018-05-15 14:19:08,728 storage          L0253 DEBUG| Checking for image available in image data path - /home/bala/avocado-fvt-wrapper/data/avocado-vt/images/hostos-ppc64le.qcow2
2018-05-15 14:19:08,728 storage          L0257 DEBUG| Copying guest image from /home/bala/avocado-fvt-wrapper/data/avocado-vt/images/hostos-ppc64le.qcow2 to /home/bala/sharing/hostos-ppc64le.qcow2
2018-05-15 14:30:18,487 unattended_insta L1027 INFO | Starting unattended install setup
2018-05-15 14:30:18,547 virt_vm          L0829 DEBUG| Generating random mac address for nic
2018-05-15 14:30:18,577 libvirt_vm       L1939 DEBUG| VM.create activating nic {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:a3:a4:a5', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-15 14:30:18,644 libvirt_vm       L0205 INFO | Libvirt VM 'vms1', driver 'qemu', uri 'qemu:///system'
2018-05-15 14:30:19,545 libvirt_vm       L0932 DEBUG| Machine types supported for hvm/ppc64le: ['pseries-2.12', 'pseries', 'ref405ep', 'sam460ex', 'virtex-ml507', 'powernv', 'ppce500', 'mpc8544ds', 'pseries-2.12-sxxm', 'bamboo', 'g3beige', 'pseries-2.10', 'prep', 'pseries-2.11', 'pseries-2.9', 'mac99', 'pseries-2.6', 'pseries-2.7', 'pseries-2.8', 'pseries-2.4', 'pseries-2.5', 'pseries-2.2', 'taihu', 'pseries-2.3', 'pseries-2.1', '40p']
2018-05-15 14:30:19,547 libvirt_vm       L0855 DEBUG| vm.add_memballoon returning:  --memballoon model=virtio
2018-05-15 14:30:19,549 libvirt_vm       L1252 DEBUG| make_create_command() setting up command for nic: {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:a3:a4:a5', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-15 14:30:19,549 libvirt_vm       L0839 DEBUG| vm.make_create_command.add_nic returning:  --network=bridge=virbr0,model=virtio,mac=52:54:00:a3:a4:a5
2018-05-15 14:30:19,549 libvirt_vm       L1945 INFO | Running libvirt command (reformatted):
2018-05-15 14:30:19,549 libvirt_vm       L1947 INFO | /usr/bin/virt-install
2018-05-15 14:30:19,550 libvirt_vm       L1947 INFO |     --connect=qemu:///system
2018-05-15 14:30:19,550 libvirt_vm       L1947 INFO |     --hvm
2018-05-15 14:30:19,550 libvirt_vm       L1947 INFO |     --accelerate
2018-05-15 14:30:19,550 libvirt_vm       L1947 INFO |     --name 'vms1'
2018-05-15 14:30:19,550 libvirt_vm       L1947 INFO |     --machine pseries
2018-05-15 14:30:19,550 libvirt_vm       L1947 INFO |     --memory=8192
2018-05-15 14:30:19,551 libvirt_vm       L1947 INFO |     --vcpu=32,sockets=1,cores=4,threads=8
2018-05-15 14:30:19,551 libvirt_vm       L1947 INFO |     --import
2018-05-15 14:30:19,551 libvirt_vm       L1947 INFO |     --nographics
2018-05-15 14:30:19,551 libvirt_vm       L1947 INFO |     --serial pty
2018-05-15 14:30:19,551 libvirt_vm       L1947 INFO |     --memballoon model=virtio
2018-05-15 14:30:19,551 libvirt_vm       L1947 INFO |     --controller type=scsi,model=virtio-scsi
2018-05-15 14:30:19,552 libvirt_vm       L1947 INFO |     --disk path=/home/bala/sharing/hostos-ppc64le.qcow2,bus=scsi,size=10,format=qcow2
2018-05-15 14:30:19,552 libvirt_vm       L1947 INFO |     --network=bridge=virbr0,model=virtio,mac=52:54:00:a3:a4:a5
2018-05-15 14:30:19,552 libvirt_vm       L1947 INFO |     --noautoconsole
2018-05-15 14:30:19,554 process          L0554 INFO | Running '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms1' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le.qcow2,bus=scsi,size=10,format=qcow2 --network=bridge=virbr0,model=virtio,mac=52:54:00:a3:a4:a5 --noautoconsole'
2018-05-15 14:30:20,640 process          L0385 DEBUG| [stderr] WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.
2018-05-15 14:30:21,676 process          L0385 DEBUG| [stdout]
2018-05-15 14:30:21,677 process          L0385 DEBUG| [stdout] Starting install...
2018-05-15 14:30:21,677 process          L0385 DEBUG| [stdout] Domain creation completed.
2018-05-15 14:30:21,822 process          L0642 INFO | Command '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms1' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le.qcow2,bus=scsi,size=10,format=qcow2 --network=bridge=virbr0,model=virtio,mac=52:54:00:a3:a4:a5 --noautoconsole' finished with 0 after 2.26421403885s
2018-05-15 14:30:21,874 utils_misc       L0681 DEBUG| waiting for domain vms1 to start (0.000012 secs)
```

**Guest clone (without import) from NFS share location:**

```
2018-05-15 15:13:04,020 storage          L0253 DEBUG| Checking for image available in image data path - /home/bala/avocado-fvt-wrapper/data/avocado-vt/images/hostos-ppc64le.qcow2
2018-05-15 15:13:04,020 storage          L0257 DEBUG| Copying guest image from /home/bala/avocado-fvt-wrapper/data/avocado-vt/images/hostos-ppc64le.qcow2 to /home/bala/sharing/hostos-ppc64le.qcow2
2018-05-15 15:18:55,601 storage          L0486 INFO | Clone master image for vms.
2018-05-15 15:18:55,602 process          L0554 INFO | Running 'cp --reflink=auto /home/bala/sharing/hostos-ppc64le.qcow2 /home/bala/sharing/hostos-ppc64le_vms1.qcow2'
2018-05-15 15:26:32,945 process          L0642 INFO | Command 'cp --reflink=auto /home/bala/sharing/hostos-ppc64le.qcow2 /home/bala/sharing/hostos-ppc64le_vms1.qcow2' finished with 0 after 457.324642897s
2018-05-15 15:26:33,364 libvirt_vm       L0205 INFO | Libvirt VM 'vms1', driver 'qemu', uri 'qemu:///system'
2018-05-15 15:26:33,421 virt_vm          L0829 DEBUG| Generating random mac address for nic
2018-05-15 15:26:33,438 libvirt_vm       L1939 DEBUG| VM.create activating nic {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:fe:ff:00', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-15 15:26:33,496 libvirt_vm       L0205 INFO | Libvirt VM 'vms1', driver 'qemu', uri 'qemu:///system'
2018-05-15 15:26:34,461 libvirt_vm       L0932 DEBUG| Machine types supported for hvm/ppc64le: ['pseries-2.12', 'pseries', 'ref405ep', 'sam460ex', 'virtex-ml507', 'powernv', 'ppce500', 'mpc8544ds', 'pseries-2.12-sxxm', 'bamboo', 'g3beige', 'pseries-2.10', 'prep', 'pseries-2.11', 'pseries-2.9', 'mac99', 'pseries-2.6', 'pseries-2.7', 'pseries-2.8', 'pseries-2.4', 'pseries-2.5', 'pseries-2.2', 'taihu', 'pseries-2.3', 'pseries-2.1', '40p']
2018-05-15 15:26:34,463 libvirt_vm       L0855 DEBUG| vm.add_memballoon returning:  --memballoon model=virtio
2018-05-15 15:26:34,465 libvirt_vm       L1252 DEBUG| make_create_command() setting up command for nic: {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:fe:ff:00', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-15 15:26:34,465 libvirt_vm       L0839 DEBUG| vm.make_create_command.add_nic returning:  --network=bridge=virbr0,model=virtio,mac=52:54:00:fe:ff:00
2018-05-15 15:26:34,465 libvirt_vm       L1945 INFO | Running libvirt command (reformatted):
2018-05-15 15:26:34,465 libvirt_vm       L1947 INFO | /usr/bin/virt-install
2018-05-15 15:26:34,466 libvirt_vm       L1947 INFO |     --connect=qemu:///system
2018-05-15 15:26:34,466 libvirt_vm       L1947 INFO |     --hvm
2018-05-15 15:26:34,466 libvirt_vm       L1947 INFO |     --accelerate
2018-05-15 15:26:34,466 libvirt_vm       L1947 INFO |     --name 'vms1'
2018-05-15 15:26:34,466 libvirt_vm       L1947 INFO |     --machine pseries
2018-05-15 15:26:34,467 libvirt_vm       L1947 INFO |     --memory=8192
2018-05-15 15:26:34,467 libvirt_vm       L1947 INFO |     --vcpu=32,sockets=1,cores=4,threads=8
2018-05-15 15:26:34,467 libvirt_vm       L1947 INFO |     --import
2018-05-15 15:26:34,467 libvirt_vm       L1947 INFO |     --nographics
2018-05-15 15:26:34,467 libvirt_vm       L1947 INFO |     --serial pty
2018-05-15 15:26:34,467 libvirt_vm       L1947 INFO |     --memballoon model=virtio
2018-05-15 15:26:34,468 libvirt_vm       L1947 INFO |     --controller type=scsi,model=virtio-scsi
2018-05-15 15:26:34,468 libvirt_vm       L1947 INFO |     --disk path=/home/bala/sharing/hostos-ppc64le_vms1.qcow2,bus=scsi,size=10,format=qcow2,cache=none
2018-05-15 15:26:34,468 libvirt_vm       L1947 INFO |     --network=bridge=virbr0,model=virtio,mac=52:54:00:fe:ff:00
2018-05-15 15:26:34,468 libvirt_vm       L1947 INFO |     --noautoconsole
2018-05-15 15:26:34,470 process          L0554 INFO | Running '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms1' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le_vms1.qcow2,bus=scsi,size=10,format=qcow2,cache=none --network=bridge=virbr0,model=virtio,mac=52:54:00:fe:ff:00 --noautoconsole'
2018-05-15 15:26:35,694 process          L0385 DEBUG| [stderr] WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.
2018-05-15 15:26:36,673 process          L0385 DEBUG| [stdout]
2018-05-15 15:26:36,673 process          L0385 DEBUG| [stdout] Starting install...
2018-05-15 15:26:36,685 process          L0385 DEBUG| [stdout] Domain creation completed.
2018-05-15 15:26:36,818 process          L0642 INFO | Command '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms1' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le_vms1.qcow2,bus=scsi,size=10,format=qcow2,cache=none --network=bridge=virbr0,model=virtio,mac=52:54:00:fe:ff:00 --noautoconsole' finished with 0 after 2.3440079689s
2018-05-15 15:26:36,871 utils_misc       L0681 DEBUG| waiting for domain vms1 to start (0.000013 secs)
```

File: virttest/storage.py [hunk 1]
```diff
@@ -227,6 +229,41 @@ def cmp(x, y):
...
+    or corrupted.
+
+    :param params: Test dict params
+    :param image_name: Master image name.
+    :param root_dir: Base directory for relative filenames.
+    :raise: TestSetupFail if image is unavailable/corrupted
+    """
+    nfs_mount_path = params.get("nfs_mount_dir")
+    image_format = params.get("image_format", "qcow2")
+    if nfs_mount_path:
```
Comment (sathnaga) [inline virttest/storage.py]: `nfs_mount_path` is checked as condn but not seeing it gets used anywhere in this method, is it hidden under any other submethods, if then can we move the logic there?

Comment (balamuruhans) [reply virttest/storage.py]: yes you are right, we have to use condition with storage_type and setup_local_nfs, I will make the respective change. Thanks for review.

**Comment by balamuruhans** [comment]
@sathnaga I have made changes as per your suggestion.

**Comment by balamuruhans** [comment]
**Tested with guest clone:**
```
2018-05-30 18:23:57,411 storage          L0252 DEBUG| Checking for image available in image data path - /home/bala/avocado-fvt-wrapper/data/avocado-vt/images/hostos-ppc64le.qcow2
2018-05-30 18:23:57,412 storage          L0256 DEBUG| Copying guest image from /home/bala/avocado-fvt-wrapper/data/avocado-vt/images/hostos-ppc64le.qcow2 to /home/bala/sharing/hostos-ppc64le.qcow2
2018-05-30 18:30:12,388 storage          L0485 INFO | Clone master image for vms.
2018-05-30 18:30:12,389 process          L0554 INFO | Running 'cp --reflink=auto /home/bala/sharing/hostos-ppc64le.qcow2 /home/bala/sharing/hostos-ppc64le_vms1.qcow2'
2018-05-30 18:40:26,132 process          L0642 INFO | Command 'cp --reflink=auto /home/bala/sharing/hostos-ppc64le.qcow2 /home/bala/sharing/hostos-ppc64le_vms1.qcow2' finished with 0 after 613.737722874s
2018-05-30 18:40:26,427 storage          L0485 INFO | Clone master image for vms.
2018-05-30 18:40:26,428 process          L0554 INFO | Running 'cp --reflink=auto /home/bala/sharing/hostos-ppc64le.qcow2 /home/bala/sharing/hostos-ppc64le_vms2.qcow2'
2018-05-30 18:50:53,374 process          L0642 INFO | Command 'cp --reflink=auto /home/bala/sharing/hostos-ppc64le.qcow2 /home/bala/sharing/hostos-ppc64le_vms2.qcow2' finished with 0 after 626.933310032s
2018-05-30 18:50:53,485 storage          L0485 INFO | Clone master image for vms.
2018-05-30 18:50:53,486 process          L0554 INFO | Running 'cp --reflink=auto /home/bala/sharing/hostos-ppc64le.qcow2 /home/bala/sharing/hostos-ppc64le_vms3.qcow2'
2018-05-30 18:58:55,526 process          L0642 INFO | Command 'cp --reflink=auto /home/bala/sharing/hostos-ppc64le.qcow2 /home/bala/sharing/hostos-ppc64le_vms3.qcow2' finished with 0 after 482.03482914s
2018-05-30 18:58:55,982 libvirt_vm       L0205 INFO | Libvirt VM 'vms1', driver 'qemu', uri 'qemu:///system'
2018-05-30 18:58:56,043 virt_vm          L0829 DEBUG| Generating random mac address for nic
2018-05-30 18:58:56,083 libvirt_vm       L1939 DEBUG| VM.create activating nic {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:ec:ed:ee', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-30 18:58:56,158 libvirt_vm       L0205 INFO | Libvirt VM 'vms1', driver 'qemu', uri 'qemu:///system'
2018-05-30 18:58:58,324 libvirt_vm       L0932 DEBUG| Machine types supported for hvm/ppc64le: ['pseries-2.12', 'pseries', 'ref405ep', 'sam460ex', 'virtex-ml507', 'powernv', 'ppce500', 'mpc8544ds', 'pseries-2.12-sxxm', 'bamboo', 'g3beige', 'pseries-2.10', 'prep', 'pseries-2.11', 'pseries-2.9', 'mac99', 'pseries-2.6', 'pseries-2.7', 'pseries-2.8', 'pseries-2.4', 'pseries-2.5', 'pseries-2.2', 'taihu', 'pseries-2.3', 'pseries-2.1', '40p']
2018-05-30 18:58:58,326 libvirt_vm       L0855 DEBUG| vm.add_memballoon returning:  --memballoon model=virtio
2018-05-30 18:58:58,328 libvirt_vm       L1252 DEBUG| make_create_command() setting up command for nic: {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:ec:ed:ee', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-30 18:58:58,328 libvirt_vm       L0839 DEBUG| vm.make_create_command.add_nic returning:  --network=bridge=virbr0,model=virtio,mac=52:54:00:ec:ed:ee
2018-05-30 18:58:58,328 libvirt_vm       L1945 INFO | Running libvirt command (reformatted):
2018-05-30 18:58:58,329 libvirt_vm       L1947 INFO | /usr/bin/virt-install
2018-05-30 18:58:58,329 libvirt_vm       L1947 INFO |     --connect=qemu:///system
2018-05-30 18:58:58,329 libvirt_vm       L1947 INFO |     --hvm
2018-05-30 18:58:58,329 libvirt_vm       L1947 INFO |     --accelerate
2018-05-30 18:58:58,329 libvirt_vm       L1947 INFO |     --name 'vms1'
2018-05-30 18:58:58,329 libvirt_vm       L1947 INFO |     --machine pseries
2018-05-30 18:58:58,330 libvirt_vm       L1947 INFO |     --memory=8192
2018-05-30 18:58:58,330 libvirt_vm       L1947 INFO |     --vcpu=32,sockets=1,cores=4,threads=8
2018-05-30 18:58:58,330 libvirt_vm       L1947 INFO |     --import
2018-05-30 18:58:58,330 libvirt_vm       L1947 INFO |     --nographics
2018-05-30 18:58:58,330 libvirt_vm       L1947 INFO |     --serial pty
2018-05-30 18:58:58,331 libvirt_vm       L1947 INFO |     --memballoon model=virtio
2018-05-30 18:58:58,331 libvirt_vm       L1947 INFO |     --controller type=scsi,model=virtio-scsi
2018-05-30 18:58:58,331 libvirt_vm       L1947 INFO |     --disk path=/home/bala/sharing/hostos-ppc64le_vms1.qcow2,bus=scsi,size=10,format=qcow2,cache=none
2018-05-30 18:58:58,331 libvirt_vm       L1947 INFO |     --network=bridge=virbr0,model=virtio,mac=52:54:00:ec:ed:ee
2018-05-30 18:58:58,331 libvirt_vm       L1947 INFO |     --noautoconsole
2018-05-30 18:58:58,333 process          L0554 INFO | Running '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms1' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le_vms1.qcow2,bus=scsi,size=10,format=qcow2,cache=none --network=bridge=virbr0,model=virtio,mac=52:54:00:ec:ed:ee --noautoconsole'
2018-05-30 18:58:59,675 process          L0385 DEBUG| [stderr] WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.
2018-05-30 18:59:01,412 process          L0385 DEBUG| [stdout]
2018-05-30 18:59:01,412 process          L0385 DEBUG| [stdout] Starting install...
2018-05-30 18:59:01,432 process          L0385 DEBUG| [stdout] Domain creation completed.
2018-05-30 18:59:01,563 process          L0642 INFO | Command '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms1' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le_vms1.qcow2,bus=scsi,size=10,format=qcow2,cache=none --network=bridge=virbr0,model=virtio,mac=52:54:00:ec:ed:ee --noautoconsole' finished with 0 after 3.22555494308s
2018-05-30 18:59:01,619 utils_misc       L0681 DEBUG| waiting for domain vms1 to start (0.000012 secs)
2018-05-30 18:59:02,028 libvirt_vm       L0205 INFO | Libvirt VM 'vms2', driver 'qemu', uri 'qemu:///system'
2018-05-30 18:59:02,088 virt_vm          L0829 DEBUG| Generating random mac address for nic
2018-05-30 18:59:02,137 libvirt_vm       L1939 DEBUG| VM.create activating nic {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:ef:f0:f1', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-30 18:59:02,215 libvirt_vm       L0205 INFO | Libvirt VM 'vms2', driver 'qemu', uri 'qemu:///system'
2018-05-30 18:59:03,182 libvirt_vm       L0932 DEBUG| Machine types supported for hvm/ppc64le: ['pseries-2.12', 'pseries', 'ref405ep', 'sam460ex', 'virtex-ml507', 'powernv', 'ppce500', 'mpc8544ds', 'pseries-2.12-sxxm', 'bamboo', 'g3beige', 'pseries-2.10', 'prep', 'pseries-2.11', 'pseries-2.9', 'mac99', 'pseries-2.6', 'pseries-2.7', 'pseries-2.8', 'pseries-2.4', 'pseries-2.5', 'pseries-2.2', 'taihu', 'pseries-2.3', 'pseries-2.1', '40p']
2018-05-30 18:59:03,183 libvirt_vm       L0855 DEBUG| vm.add_memballoon returning:  --memballoon model=virtio
2018-05-30 18:59:03,184 libvirt_vm       L1252 DEBUG| make_create_command() setting up command for nic: {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:ef:f0:f1', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-30 18:59:03,185 libvirt_vm       L0839 DEBUG| vm.make_create_command.add_nic returning:  --network=bridge=virbr0,model=virtio,mac=52:54:00:ef:f0:f1
2018-05-30 18:59:03,185 libvirt_vm       L1945 INFO | Running libvirt command (reformatted):
2018-05-30 18:59:03,185 libvirt_vm       L1947 INFO | /usr/bin/virt-install
2018-05-30 18:59:03,185 libvirt_vm       L1947 INFO |     --connect=qemu:///system
2018-05-30 18:59:03,186 libvirt_vm       L1947 INFO |     --hvm
2018-05-30 18:59:03,186 libvirt_vm       L1947 INFO |     --accelerate
2018-05-30 18:59:03,186 libvirt_vm       L1947 INFO |     --name 'vms2'
2018-05-30 18:59:03,186 libvirt_vm       L1947 INFO |     --machine pseries
2018-05-30 18:59:03,186 libvirt_vm       L1947 INFO |     --memory=8192
2018-05-30 18:59:03,186 libvirt_vm       L1947 INFO |     --vcpu=32,sockets=1,cores=4,threads=8
2018-05-30 18:59:03,187 libvirt_vm       L1947 INFO |     --import
2018-05-30 18:59:03,187 libvirt_vm       L1947 INFO |     --nographics
2018-05-30 18:59:03,187 libvirt_vm       L1947 INFO |     --serial pty
2018-05-30 18:59:03,187 libvirt_vm       L1947 INFO |     --memballoon model=virtio
2018-05-30 18:59:03,191 libvirt_vm       L1947 INFO |     --controller type=scsi,model=virtio-scsi
2018-05-30 18:59:03,191 libvirt_vm       L1947 INFO |     --disk path=/home/bala/sharing/hostos-ppc64le_vms2.qcow2,bus=scsi,size=10,format=qcow2,cache=none
2018-05-30 18:59:03,191 libvirt_vm       L1947 INFO |     --network=bridge=virbr0,model=virtio,mac=52:54:00:ef:f0:f1
2018-05-30 18:59:03,192 libvirt_vm       L1947 INFO |     --noautoconsole
2018-05-30 18:59:03,194 process          L0554 INFO | Running '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms2' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le_vms2.qcow2,bus=scsi,size=10,format=qcow2,cache=none --network=bridge=virbr0,model=virtio,mac=52:54:00:ef:f0:f1 --noautoconsole'
2018-05-30 18:59:04,021 process          L0385 DEBUG| [stderr] WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.
2018-05-30 18:59:04,464 process          L0385 DEBUG| [stdout]
2018-05-30 18:59:04,464 process          L0385 DEBUG| [stdout] Starting install...
2018-05-30 18:59:04,465 process          L0385 DEBUG| [stdout] Domain creation completed.
2018-05-30 18:59:04,605 process          L0642 INFO | Command '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms2' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le_vms2.qcow2,bus=scsi,size=10,format=qcow2,cache=none --network=bridge=virbr0,model=virtio,mac=52:54:00:ef:f0:f1 --noautoconsole' finished with 0 after 1.40749597549s
2018-05-30 18:59:04,651 utils_misc       L0681 DEBUG| waiting for domain vms2 to start (0.000010 secs)
2018-05-30 18:59:04,911 libvirt_vm       L0205 INFO | Libvirt VM 'vms3', driver 'qemu', uri 'qemu:///system'
2018-05-30 18:59:04,963 virt_vm          L0829 DEBUG| Generating random mac address for nic
2018-05-30 18:59:05,002 libvirt_vm       L1939 DEBUG| VM.create activating nic {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:f2:f3:f4', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-30 18:59:05,069 libvirt_vm       L0205 INFO | Libvirt VM 'vms3', driver 'qemu', uri 'qemu:///system'
2018-05-30 18:59:05,972 libvirt_vm       L0932 DEBUG| Machine types supported for hvm/ppc64le: ['pseries-2.12', 'pseries', 'ref405ep', 'sam460ex', 'virtex-ml507', 'powernv', 'ppce500', 'mpc8544ds', 'pseries-2.12-sxxm', 'bamboo', 'g3beige', 'pseries-2.10', 'prep', 'pseries-2.11', 'pseries-2.9', 'mac99', 'pseries-2.6', 'pseries-2.7', 'pseries-2.8', 'pseries-2.4', 'pseries-2.5', 'pseries-2.2', 'taihu', 'pseries-2.3', 'pseries-2.1', '40p']
2018-05-30 18:59:05,973 libvirt_vm       L0855 DEBUG| vm.add_memballoon returning:  --memballoon model=virtio
2018-05-30 18:59:05,975 libvirt_vm       L1252 DEBUG| make_create_command() setting up command for nic: {'netdst': 'virbr0', 'ip': None, 'nic_name': 'nic1', 'mac': '52:54:00:f2:f3:f4', 'nettype': 'bridge', 'nic_model': 'virtio', 'g_nic_name': None}
2018-05-30 18:59:05,975 libvirt_vm       L0839 DEBUG| vm.make_create_command.add_nic returning:  --network=bridge=virbr0,model=virtio,mac=52:54:00:f2:f3:f4
2018-05-30 18:59:05,975 libvirt_vm       L1945 INFO | Running libvirt command (reformatted):
2018-05-30 18:59:05,975 libvirt_vm       L1947 INFO | /usr/bin/virt-install
2018-05-30 18:59:05,976 libvirt_vm       L1947 INFO |     --connect=qemu:///system
2018-05-30 18:59:05,976 libvirt_vm       L1947 INFO |     --hvm
2018-05-30 18:59:05,976 libvirt_vm       L1947 INFO |     --accelerate
2018-05-30 18:59:05,976 libvirt_vm       L1947 INFO |     --name 'vms3'
2018-05-30 18:59:05,976 libvirt_vm       L1947 INFO |     --machine pseries
2018-05-30 18:59:05,977 libvirt_vm       L1947 INFO |     --memory=8192
2018-05-30 18:59:05,977 libvirt_vm       L1947 INFO |     --vcpu=32,sockets=1,cores=4,threads=8
2018-05-30 18:59:05,977 libvirt_vm       L1947 INFO |     --import
2018-05-30 18:59:05,977 libvirt_vm       L1947 INFO |     --nographics
2018-05-30 18:59:05,977 libvirt_vm       L1947 INFO |     --serial pty
2018-05-30 18:59:05,977 libvirt_vm       L1947 INFO |     --memballoon model=virtio
2018-05-30 18:59:05,978 libvirt_vm       L1947 INFO |     --controller type=scsi,model=virtio-scsi
2018-05-30 18:59:05,978 libvirt_vm       L1947 INFO |     --disk path=/home/bala/sharing/hostos-ppc64le_vms3.qcow2,bus=scsi,size=10,format=qcow2,cache=none
2018-05-30 18:59:05,978 libvirt_vm       L1947 INFO |     --network=bridge=virbr0,model=virtio,mac=52:54:00:f2:f3:f4
2018-05-30 18:59:05,978 libvirt_vm       L1947 INFO |     --noautoconsole
2018-05-30 18:59:05,980 process          L0554 INFO | Running '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms3' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le_vms3.qcow2,bus=scsi,size=10,format=qcow2,cache=none --network=bridge=virbr0,model=virtio,mac=52:54:00:f2:f3:f4 --noautoconsole'
2018-05-30 18:59:06,691 process          L0385 DEBUG| [stderr] WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.
2018-05-30 18:59:07,155 process          L0385 DEBUG| [stdout]
2018-05-30 18:59:07,156 process          L0385 DEBUG| [stdout] Starting install...
2018-05-30 18:59:07,156 process          L0385 DEBUG| [stdout] Domain creation completed.
2018-05-30 18:59:07,302 process          L0642 INFO | Command '/usr/bin/virt-install --connect=qemu:///system --hvm --accelerate --name 'vms3' --machine pseries --memory=8192 --vcpu=32,sockets=1,cores=4,threads=8 --import --nographics --serial pty --memballoon model=virtio --controller type=scsi,model=virtio-scsi --disk path=/home/bala/sharing/hostos-ppc64le_vms3.qcow2,bus=scsi,size=10,format=qcow2,cache=none --network=bridge=virbr0,model=virtio,mac=52:54:00:f2:f3:f4 --noautoconsole' finished with 0 after 1.31778001785s
2018-05-30 18:59:07,349 utils_misc       L0681 DEBUG| waiting for domain vms3 to start (0.000010 secs)


```

**Pull request merged**